ComputeClient/Compute.Client.UnitTests/Server20/ServerAccessorTests.cs

```diff
@@ -10,55 +10,55 @@
 
 namespace Compute.Client.UnitTests.Server20
 {
-	[TestClass]
-	public class ServerAccessorTests : BaseApiClientTestFixture
-	{
-		[TestMethod]
-		public async Task GetServers_ReturnsResponse()
-		{
-			requestsAndResponses.Add(ApiUris.MyAccount,RequestFileResponseType.AsGoodResponse("GetMyAccountDetails.xml"));
-			requestsAndResponses.Add(ApiUris.GetMcp2Servers(accountId), RequestFileResponseType.AsGoodResponse("GetServersResponse.xml"));
+    [TestClass]
```

Comment:
formated file using Visual Studio 2017 - real changes in this file start on line 266